src/parser/monk/mistweaver/modules/spells/ThunderFocusTea.tsx

```diff
@@ -82,7 +84,7 @@ class ThunderFocusTea extends Analyzer {
   }
 
   renderCastRatioChart() {
-    const items = [
+    const items: {}[] = [
```

Comment:
Lint screaming about this; i would say just let typescript infer the type, it's a const already